x-pack/test/security_solution_cypress/cypress/e2e/investigations/timeline_templates/export.cy.ts

```diff
@@ -16,9 +16,11 @@ import {
 import { TIMELINE_TEMPLATES_URL } from '../../../urls/navigation';
 import { createTimelineTemplate } from '../../../tasks/api_calls/timelines';
 import { searchByTitle } from '../../../tasks/table_pagination';
+import { deleteTimelines } from '../../../tasks/api_calls/common';
 
 describe('Export timelines', { tags: ['@ess', '@serverless'] }, () => {
-  before(() => {
+  beforeEach(() => {
+    deleteTimelines();
```

Comment:
@MadameSheema just for my own knowledge, what's the point of deleting timelines here? There is only a single test, and unless I'm mistaken this should run on a fresh instance of everything so no timeline should be there. In case of a failing test and a retry, I'm not sure have 2 timelines would make the test run differently. Let me try locally and see!

Reply:
That method per the way was implemented deletes also templates and we are creating one. Even if is only one test, as there people copy-pasting when they create something new, is good to have it if it is a good practice we want to follow.